cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -1,79 +1,56 @@
+  radioButtonInputByPosition(positionOfElement(position)).should('have.attr', 'value').should('contain', text);
 });
 
-Then('{string} RadioButton component is disabled', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('be.disabled')
-    .and('have.attr', 'disabled');
+Then('{string} RadioButton component is disabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.attr', 'disabled');
 });
 
-Then('{string} RadioButton component is enabled', (radioButtonName) => {
-  cy.radioButtonComponent(radioButtonName).should('not.be.disabled').and('not.have.attr', 'disabled');
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('not.be.disabled')
+Then('{string} RadioButton component is enabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('not.be.disabled')
     .and('not.have.attr', 'disabled');
 });
 
-Then('{string} RadioButton is set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, 'div:nth-child(2) input').should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('div:nth-child(2) input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton is not set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is not set to reverse', (position) => {
+  radioButtonInputByPosition(positionOfElement(position)).should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton size on preview is set to {string}', (radioButtonName, size) => {
+Then('{string} RadioButton size on preview is set to {string}', (position, size) => {
   if (size === 'small') {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '16px')
+    radioButtonInputByPosition(positionOfElement(position)).should('have.css', 'width', '16px')
       .and('have.css', 'height', '16px');
   } else {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '24px')
+    radioButtonInputByPosition(positionOfElement(position)).should('have.css', 'width', '24px')
       .and('have.css', 'height', '24px');
   }
 });
 
-Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (radioButtonName, marginLeft, marginRight) => {
-  cy.radioButton(radioButtonName, 'div div span[data-element="help"]').should('have.css', 'margin-left', marginLeft)
+Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (position, marginLeft, marginRight) => {
+  radioButtonFieldHelp(positionOfElement(position)).should('have.css', 'margin-left', marginLeft)
```

Comment:
```suggestion
  radioButtonFieldHelpNoIFrame(positionOfElement(position)).should('have.css', 'margin-left', marginLeft)
```

Reply:
as we discussed if we using it in `noIframe` the name is e.g `radioButtonFieldHelp`.`